build.gradle

```diff
@@ -0,0 +1,51 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+
+    dependencies {
+        classpath 'com.palantir.gradle.externalpublish:gradle-external-publish-plugin:1.4.0'
+        classpath 'com.palantir.gradle.gitversion:gradle-git-version:0.12.3'
+    }
+}
+
+/**
+ * Previously we used a shell script to publish the distribution to bintray. With the migration to maven-central, we
+ * need to leverage more tooling coming from 'gradle-external-publish-plugin'. Based on that, this repo is currently
+ * in an awkward split-mode. This Gradle setup is just for publishing the dist to maven-central. The actual build and
+ * compilation is still done through yarn (see 'package.json').
+ * TODO(fawind): Move this project fully to Gradle (i.e. using 'gradlets/gradle-typescript').
+ */
+apply plugin: 'com.palantir.external-publish-dist'
+apply plugin: 'com.palantir.git-version'
+
+group 'com.palantir.conjure.typescript'
+version gitVersion()
+
+task verifyBinariesExist() {
+    [file('dist/bin/conjure-typescript'), file('dist/bin/conjure-typescript.bat')]
```

Comment:
I think you need to put this in a `doFirst`/`doLast` or this code will run at configuration time rather than as a task action?

Reply:
Good catch, updated! It didn't fail because we generate these files before we invoke gradle but no need to do this at configuration time.